src/server/zset_family.cc

```diff
@@ -48,6 +54,30 @@ using MScoreResponse = std::vector<std::optional<double>>;
 using ScoredMember = std::pair<std::string, double>;
 using ScoredArray = std::vector<ScoredMember>;
 
+struct GeoPoint {
+  double longitude;
+  double latitude;
+  double dist;
+  double score;
+  std::string member;
+  GeoPoint() : longitude(0.0), latitude(0.0), dist(0.0), score(0.0), member(""){};
+  GeoPoint(double _longitude, double _latitude, double _dist, double _score,
+           const std::string& _member)
+      : longitude(_longitude), latitude(_latitude), dist(_dist), score(_score), member(_member){};
+};
+using GeoArray = std::vector<GeoPoint>;
+
+struct GeoSearchOpts {
+  double conversion = 0;
+  uint64_t count = 0;
+  bool asc = 0;
+  bool desc = 0;
```

Comment:
Maybe add an enum like:

`enum class Sorting { kUnsorted, kAsc, kDesc };`

And replace these members with `Sorting sorting;`? 

This would be easier to read than a bunch of booleans, and also will only represent valid cases (with this code - both could theoretically be true, which doesn't make sense)